data/manhuagui

```diff
@@ -0,0 +1,2 @@
+manhuagui.com
+hamreus.com
```

Comment:
![image](https://user-images.githubusercontent.com/32576256/146644062-78dc913d-4e68-416a-aac6-2f3013af4aef.png)

what the heck is this.